karma.conf.js

```diff
@@ -4,18 +4,30 @@ if (!process.env.TRAVIS){
     process.env.CHROME_BIN = require('puppeteer').executablePath();
 }
 
+function getTestFiles(){
```

Comment:
Is it possible to move out those helper functions? and are you sure the minified file still does get ignored from browserify?

Reply:
These are only used here. I think moving them out would reduce a clarity a bit / make the logic harder to follow. What is your reasoning fo 

> are you sure the minified file still does get ignored from browserify?

Yes! You should check this yourself, its the whole point of the test.

Reply:
> What is your reasoning for putting outside? And where?

To have all helpers in one place which do use the environment vars this would increase the clarity in this case. But yes it does otherwise decrease the readability in the karma file because you have to follow the execution path over multiple files. 
The conclusion from my side is: Let us keep it as it is now and if the complexity with the environment variables would increase over time can we look around for another more "centralized" structure.

Reply:
> Yes! You should check this yourself, its the whole point of the test.

I will run it on the finish. :)

Reply:
> if the complexity with the environment variables would increase over time can we look around for another more "centralized" structure.

Yes, definitely.